spec/graphql/schema/dynamic_members_spec.rb

```diff
@@ -439,7 +439,7 @@ def legacy_schema_sdl
       MultifieldSchema::Query.get_field("f1")
     end
 
-    expected_message = "Found two visible definitions for `Query.f1`: #<MultifieldSchema::BaseField Query.f1: String>, #<MultifieldSchema::BaseField Query.f1: Int>"
+    expected_message = "Found two visible type definitions for `Query.f1`: #<MultifieldSchema::BaseField Query.f1: String>, #<MultifieldSchema::BaseField Query.f1: Int>"
```

Comment:
I can either do that or introduce a new error for `types`. We can have `DuplicateFieldNameError` and `DuplicateFieldTypeError`. I prefer the second since it's more granular and explicit, but ok with the first approach.

Reply:
Either approach suits me! :+1: